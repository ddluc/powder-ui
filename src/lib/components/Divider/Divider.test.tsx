import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Divider, DividerProps } from './index';
import withTheme from '../../../../test/withTheme';

describe('Divider', () => {
  afterEach(cleanup);

  const args: DividerProps = {
    type: 'horizontal',
    solid: true
  };

  it('should render', () => {
    const result = withTheme(<Divider {...args} />);
    const component = result.container.firstChild;
    expect(component).toMatchSnapshot();
  });
});
