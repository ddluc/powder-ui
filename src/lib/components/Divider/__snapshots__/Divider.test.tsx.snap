// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should render 1`] = `
.c0 {
  position: relative;
  border-bottom: solid 1px;
  border-color: #E1EFED;
  border-top: none;
  border-right: none;
  border-left: none;
  height: 1px;
  width: 100%;
  margin-top: undefined;
  margin-bottom: undefined;
  margin-left: 0px;
  margin-right: auto;
}

<hr
  class="c0"
  type="horizontal"
/>
`;
